boilerplate/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM node:8.9-alpine
+
+# Initialize app directory
+ENV APP_DIR "/usr/app"
+ENV PYTHONPATH="/:${PYTHONPATH}"
+
+RUN mkdir -p $APP_DIR
+WORKDIR $APP_DIR
+
+#PM2 will be used as PID 1 process
+RUN npm install -g pm2@1.1.3
+
+# for converting files to unix(CRLF)
```

Comment:
This is up for debate, but in my opinion, it would be cleaner to [let git handle this](https://help.github.com/en/articles/configuring-git-to-handle-line-endings) and not install a separate program in the Docker container.

Reply:
> This is up for debate, but in my opinion, it would be cleaner to [let git handle CRLF conversion](https://help.github.com/en/articles/configuring-git-to-handle-line-endings) and not install a separate program in the Docker container.

I agree. This should not be in the Dockerfile. Please remove this and make this change in git.

Reply:
the problem is when I copy and paste files from Windows into linux containers it automatically adds `^M` at the end of every line in every file. that's why I have the converter here